dependent-usage-analyzer/index.js

```diff
@@ -30,6 +30,7 @@ function getPackageInfo(dir) {
       version: dependencies && dependencies['@edx/paragon'] ? dependencies['@edx/paragon'].version : 'unknown',
       name,
       repository,
+      folderName: dir.split('/').pop(),
```

Comment:
Our `name` field in package json across projects is inconsistent. Folder names are better.